instrumentation/runtime-telemetry/runtime-telemetry-java8/library/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
 
 dependencies {
   implementation(project(":instrumentation-api"))
-  implementation("io.opentelemetry:opentelemetry-extension-incubator")
+  implementation("io.opentelemetry:opentelemetry-api-incubator")
```

Comment:
is this dependency needed at all?

Reply:
yes, for `EventLogger`

Reply:
this module does not use `EventLogger` as far as I can tell

Reply:
right, mixed that up with another module - removed